Makefile

```diff
@@ -249,14 +239,91 @@ ifeq ($(OUTPUT_CSV), true)
   $(shell touch .update.timestamp)
 endif
 
+
+##
+##	StatusQ
+##
+
+ifneq ($(detected_OS),Windows)
+ STATUSQ := bin/StatusQ/libStatusQ.$(LIBSTATUS_EXT)
+else
+ STATUSQ := bin/StatusQ/StatusQ.$(LIBSTATUS_EXT)
+ STATUSQ_CMAKE_CONFIG_PARAMS := -T"v141" -A x64
+ STATUSQ_CMAKE_BUILD_PARAMS := --config Release
```

Comment:
At first because `--config` is only used on multiconfiguration builds, which is only Windows for us.
But I agree that it doesn't hurt anyway to add it anywhere. And it's more consistent also.

`CMAKE_BUILD_TYPE` is `Release` by default, so I didn't add it.

Reply:
It's worth noting that developers may choose their preferred configuration, and in my case, I often opt for Ninja Multi-Config on Linux.

Reply:
I added `--config Release` 👌